File: fizzbuzz.c
#include<stdio.h>

int main(void){
  int num = 1;
  while(1){
    printf("%d",(num%3==0 && num%7==0) ? "FizzBuzz!\n" : (num%3==0 ? "Fizz\n" : (num%7==0 ? "Buzz" : num)));
    num++;
  }
}

File: fizzbuzz.js
var num = 1;
while(1){
  if(num%3==0){
    document.write(num%7==0 ? "FizzBuzz!<br />" : "Fizz<br />");
  } else {
    document.write(num%7==0 ? "Buzz<br />" : num+"<br />");
  }
  num++;
  if(num==100) break;
}
